storage/go.mod

```diff
@@ -0,0 +1,22 @@
+module github.com/Scalingo/go-utils/storage
+
+go 1.14
+
+require (
+	github.com/Scalingo/go-utils/logger v1.0.0 // indirect
+	github.com/aws/aws-sdk-go v1.35.9
+	github.com/aws/aws-sdk-go-v2 v0.27.0
+	github.com/aws/aws-sdk-go-v2/credentials v0.1.2
+	github.com/aws/aws-sdk-go-v2/feature/s3/manager v0.1.0
+	github.com/aws/aws-sdk-go-v2/service/internal/accept-encoding v0.2.0 // indirect
+	github.com/aws/aws-sdk-go-v2/service/s3 v0.27.0
+	github.com/awslabs/smithy-go v0.2.0
```

Comment:
Update to v0.3.0 which just got released so that we are up-to-date for the v1 release of the storage lib?

Reply:
Done